token-lending/sdk/src/state/reserve.rs

```diff
@@ -322,31 +322,42 @@ impl Reserve {
         let repay_amount;
         let withdraw_amount;
 
-        // Close out obligations that are too small to liquidate normally
-        if liquidity.borrowed_amount_wads < LIQUIDATION_CLOSE_AMOUNT.into() {
-            // settle_amount is fixed, calculate withdraw_amount and repay_amount
-            settle_amount = liquidity.borrowed_amount_wads;
-
+        // do a full liquidation if the market value of the borrow is less than one.
+        if liquidity.market_value <= Decimal::one() {
             let liquidation_value = liquidity.market_value.try_mul(bonus_rate)?;
```

Comment:
maybe this should be liquidity.borrow_amount_wads.try_ceil_u64()? then convert that amount into a dollar value